crates/collab/src/llm.rs

```diff
@@ -440,10 +440,6 @@ async fn predict_edits(
     _country_code_header: Option<TypedHeader<CloudflareIpCountryHeader>>,
     Json(params): Json<PredictEditsParams>,
 ) -> Result<impl IntoResponse> {
-    if !claims.is_staff {
-        return Err(anyhow!("not found"))?;
-    }
-
```

Comment:
The task in the channel notes only mentioned removing this check, but don't we need to check a feature flag or something?